### PR TITLE
deduplicate jettons addresses while merge

### PR DESCRIPTION
**Pull request by erokhinav**
Please make sure you change the original ones `.yaml` fiels in directories `accounts/`, `collections/` or  `jettons/` and do not touch automatically generated .json files in the repository root.

Пожалуйста, убедитесь, что вы изменяете исходные `.yaml` файлы в директориях  `accounts/`, `collections/` или  `jettons/` и не трогаете автоматически сгенерированные файлы `.json` в корне репозитория.

### CODE REVIEW

**Comment by fati1365** [comment]
نمیدونم چرا همه چی قاطی شده

**Comment by Arslan-007ikram** [comment]
Hello, why are you worried?

чт, 3 окт. 2024 г., 11:38 fati1365 ***@***.***>:

**Comment by Mamad3426m** [comment]
چه کاری باعث نگرانی شده

**Review by naxoootcnjncgjnnxfhjjv — COMMENTED** [review]
5cf28428dd36680371f7afe36a773c70bb4f3a37

File: generator.py
Comment (naxoootcnjncgjnnxfhjjv) [inline generator.py:1]: generator.py

Comment (naxoootcnjncgjnnxfhjjv) [reply generator.py:1]: generator.py

Comment (jwad36608) [inline generator.py:1]: generator.py

**Review by Hamaster2 — APPROVED** [review]

File: generator.py
Comment (naxoootcnjncgjnnxfhjjv) [reply generator.py:1]: generator.py

**Review by naxoootcnjncgjnnxfhjjv — COMMENTED** [review]
python3 generator.py

**Comment by naxoootcnjncgjnnxfhjjv** [comment]
This pull request includes changes to the `merge_jettons` function in `generator.py` to improve address handling and validation. The most important changes include adding a dictionary to track existing addresses and ensuring no duplicate addresses are present.

Validation improvements:

* [`generator.py`](diffhunk://#diff-dbb052c2e3632b3de232373278391035edd31dcbddc9c56dac6e3cf04c97abc6R59-R72): Added a dictionary `already_exist_address` to track normalized addresses and ensure no duplicate addresses are present, raising an exception if a duplicate is found.

File: generator.py
Comment (ahora400) [reply generator.py:1]: چی شده

**Pull request merged**